### PR TITLE
(#110) added documentation for sakuli cli

### PR DESCRIPTION
**Pull request by cliffle**

### CODE REVIEW

File: content/docs/cli_commands.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+{{<highlight bash>}}
+sakuli enable-enterprise
+
+
+Configures and enables enterprise features
+{{</highlight>}}
+
+### sakuli enable-typescript
+{{<highlight bash>}}
+sakuli enable-typescript [project]
```
Comment (svettwer) [inline content/docs/cli_commands.md]: If `project` is required, it should be noted without braces.
```suggestion
sakuli enable-typescript project
```

File: content/docs/cli_commands.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Enables Typescript support for the provided project
+
+
+Positionals:
+  project               Path to the project, can be absolute or relative to $PWD    [required]
+{{</highlight>}}
+
+### sakuli encrypt
+{{<highlight bash>}}
+sakuli encrypt [secret]
```
Comment (svettwer) [inline content/docs/cli_commands.md]: If `secret` is required, it should be noted without braces.
```suggestion
sakuli encrypt secret
```

File: content/docs/cli_commands.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+Positionals:
+  secret                The secret to encrypt                                   [required]
+  
+Options:
+  --masterkey           The masterkey used for encryption
+{{</highlight>}}
+
+### sakuli migrate
+{{<highlight bash>}}
+sakuli migrate [path]
```
Comment (svettwer) [inline content/docs/cli_commands.md]: If `project` is required, it should be noted without braces.
```suggestion
sakuli migrate path
```

File: content/docs/cli_commands.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+
+Positional
+  path                  path to a legacy suite                                  [required]
+{{</highlight>}}
+
+
+### sakuli run
+{{<highlight bash>}}
+sakuli run [path]
```
Comment (svettwer) [inline content/docs/cli_commands.md]: If `project` is required, it should be noted without braces.
```suggestion
sakuli run path
```

**Comment by tnobody** [comment]
Some remarks:
First of all: It looks pretty cool with the bash dump :)

Some points I would change/extend:
- We can remove the link list at the top because the TOC is already displayed on the right side
- We should add some text to each command. At least one paragraph.
  - e.g. `sakuli create project` short describtion how to create a project from scratch and on description on how to create a new testsuite in an already existing project
  - `sakuli create masterkey` -> as a user I want to know what the master key is and how I can utilize it (add some code sample with a pseudo login to some service e.g.)
  - `enable-entprise` at least point to the enterprise page where the command usage is described
  - `enable-typescript` Maybe we should skip this until the feature is complete

File: content/docs/cli_commands.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+weight: 0
+---
+
+* [sakuli create project](#sakuli-create-project)
+* [sakuli create masterkey](#sakuli-create-masterkey)
+* [sakuli enable-enterprise](#sakuli-enable-enterprise)
+* [sakuli enable-typescript](#sakuli-enable-typescript)
+* [sakuli encrypt](#sakuli-encrypt)
+* [sakuli migrate](#sakuli-migrate)
+* [sakuli run](#sakuli-run)
```
Comment (svettwer) [inline content/docs/cli_commands.md]: This listing can be removed as it is generated in the right navi

**Pull request merged**